examples/uplink_set.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+)
+
+func main() {
+	var (
+		ClientOV        *ov.OVClient
+		existing_uplink = "upset66"
+		new_uplink      = "upset77"
+		upd_uplink      = "upset88"
+		del_uplink      = "upset88"
+	)
+	ovc := ClientOV.NewOVClient(
+		os.Getenv("ONEVIEW_OV_USER"),
+		os.Getenv("ONEVIEW_OV_PASSWORD"),
+		os.Getenv("ONEVIEW_OV_DOMAIN"),
+		os.Getenv("ONEVIEW_OV_ENDPOINT"),
+		false,
+		800)
+	ovVer, _ := ovc.GetAPIVersion()
+	fmt.Println(ovVer)
+
+	fmt.Println("#................... Uplink-Set by Name ...............#")
+	uplink_set, err := ovc.GetUplinkSetByName(existing_uplink)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println(uplink_set)
+	}
+
+	sort := "name:desc"
+	uplinkset_list, err := ovc.GetUplinkSets("", sort)
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		fmt.Println("# ................... Uplink-Set List .................#")
+		for i := 0; i < len(uplinkset_list.Members); i++ {
+			fmt.Println(uplinkset_list.Members[i].Name)
+		}
+	}
+
+	networkUris := new([]utils.Nstring)
+	*networkUris = append(*networkUris, utils.NewNstring("/rest/ethernet-networks/cbde97d0-c8f1-4aba-aa86-2b4e5d080401"))
+	//      Using make as the [] is going as null
+	fcNetworkUris := make([]utils.Nstring, 0)
+	fcoeNetworkUris := make([]utils.Nstring, 0)
+	portConfigInfos := make([]ov.PortConfigInfos, 0)
+
+	uplinkSet := ov.UplinkSet{Name: "upset77", LogicalInterconnectURI: utils.NewNstring("/rest/logical-interconnects/d4468f89-4442-4324-9c01-624c7382db2d"), NetworkURIs: *networkUris, FcNetworkURIs: fcNetworkUris, FcoeNetworkURIs: fcoeNetworkUris, PortConfigInfos: portConfigInfos, ConnectionMode: "Auto", NetworkType: "Ethernet", EthernetNetworkType: "Tagged", Type: "uplink-setV4", ManualLoginRedistributionState: "NotSupported"}
```

Comment:
We can make this struct parameters so as to increase the readability for the user.